pkg/kritis/util/whitelist_test.go

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright 2018 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package util
+
+import (
+	"github.com/grafeas/kritis/pkg/kritis/testutil"
+	"testing"
+)
+
+func Test_CheckGlobalWhitelist(t *testing.T) {
+	tests := []struct {
+		name     string
+		images   []string
+		expected bool
+	}{
+		{
+			name: "images in whitelist",
+			images: []string{
+				"gcr.io/kritis-project/kritis-server:tag",
+				"gcr.io/kritis-project/kritis-server@sha256:0000000000000000000000000000000000000000000000000000000000000000",
+			},
+			expected: true,
+		},
+		{
+			name: "some images not whitelisted",
+			images: []string{
```

Comment:
What is the difference in tc 1 and tc 2?

Reply:
the second test case has an image that isn't globally whitelisted, so checks that the function returns false

Reply:
Maybe i am missing something. Arent they the same image names?

Reply:
Oh haha yah I can see how it's not the clearest, the second image is in project kritis instead of kritis-project

Reply:
ah ok. Can you instead add some/image@sha:00000 ?